rest/src/main/java/org/jboss/pnc/rest/endpoints/internal/DeliverableAnalysisEndpointImpl.java

```diff
@@ -38,9 +47,27 @@ public class DeliverableAnalysisEndpointImpl implements DeliverableAnalysisEndpo
     @Inject
     private DeliverableAnalyzerOperationMapper deliverableAnalyzerOperationMapper;
 
+    @Inject
+    private ManagedExecutorService executorService;
+
     @Override
     public void completeAnalysis(AnalysisResult response) {
-        resultProcessor.completeAnalysis(transformToModelAnalysisResult(response));
+        executorService.execute(() -> {
+            ResultStatus result;
+            try {
+                resultProcessor.completeAnalysis(transformToModelAnalysisResult(response));
+                result = ResultStatus.SUCCESS;
+            } catch (RuntimeException e) {
+                log.error(
+                        String.format(
+                                "Storing results of deliverable operation with id=%s failed",
+                                response.getOperationId()),
+                        e);
```

Comment:
By default, SLF4J will use the latest parameter as a candidate for Throwable. If the provided parameter is an exception, SLF4J will print the complete stack trace in the log output.

For example, for a given logline:

log.info("App is running at {}, zone = {}, java version = {}, java vm = {}", LocalDateTime.now(), ZonedDateTime.now()
  .getZone(), System.getProperty("java.version"), System.getProperty("java.vm.name"), exceptionCause);
Copy
```suggestion
                log.error("Storing results of deliverable operation with id={} failed", response.getOperationId(), e);
```